package.json

```diff
@@ -3,10 +3,14 @@
 		"eslint": "^5.15.3",
 		"eslint-config-liferay": "^3.0.0",
 		"lerna": "^2.4.0",
-		"prettier-eslint-cli": "^4.7.1"
+		"prettier": "^1.16.4"
 	},
 	"scripts": {
-		"format": "prettier-eslint \"./**/*.js\" --write",
-		"lerna": "npm install && lerna bootstrap"
+		"fix": "npm run format && npm run lint:fix",
+		"format:check": "prettier --list-different '**/*.js'",
+		"format": "prettier --write '**/*.js'",
+		"lerna": "npm install && lerna bootstrap",
+		"lint": "eslint '**/*.js'",
+		"lint:fix": "eslint --fix '**/*.js'"
```

Comment:
How do you feel about changing the names up a bit? Using `format`, `fix`, and `lint:fix` are all a bit confusing without looking at what they actually do, especially `format` and `fix`.

I'm thinking something like 
```json
{
	"format": "npm run prettier && npm run lint:fix",
	"prettier:check": "prettier --list-different '**/*.js'",
	"prettier": "prettier --write '**/*.js'",
	"lerna": "npm install && lerna bootstrap",
	"lint": "eslint '**/*.js'",
	"lint:fix": "eslint --fix '**/*.js'"
}
```

Reply:
I'm open to changing the names but I can give you some insight into why they are named they are:

- "format" and "format:check" instead of "prettier" and "prettier:check" in order to say what they do instead of how they do it (an implementation detail); ie. the same reason that we have "test" script that runs `jest` instead of creating a "jest" one. I know it's unlikely we'll ever use anything but Prettier for formatting, but certainly in the case of testing and linting we have legacy projects that use other tools (Karma, Mocha etc).
- "lint" and "lint:fix" for similar reasons.

So I think those are good. "fix", however, is the worst name: I only added it because you wanted one command that did both linting and fixing. I don't think "format" is a good name for that task, because the whole point of this change is to decoupling the concerns of formatting and linting. Do you have any other suggestions for what a good name for this "fix all the things task"?

Reply:
Ah I see, I had figured you wanted to go the route of "say what they do instead of how they do it", which I am good with, I think it was the "fix" one that through me off when it is coupled with the rest.

Since the "fix" script was added for my concern of doing it all in one command, I'd say just go ahead and remove it. I can just manually run those myself with `npm run format && npm run lint:fix`.

TLDR: Remove fix script, keep the rest as is.

Reply:
> TLDR: Remove fix script, keep the rest as is.

Disappointed that you don't like the name, "do:not:use:or:you:will:be:fired". 😢

Reply:
😂

First I thought, maybe call it "fix:all", but then I realized I just don't really need it and I'm always a big fan of removing code rather than adding